src/HHbbVV/processors/bbVVSkimmer.py

```diff
@@ -188,6 +188,7 @@ def process(self, events: ak.Array):
 
         isData = "JetHT" in dataset
         isQCD = "QCD" in dataset
+        isVBFSearch = True
```

Comment:
Just noticed this too - can you turn this into an argument to the skimmer instead? You'll have to propagate it in few places too unfortunately:

[1] https://github.com/rkansal47/HHbbVV/blob/main/src/run_utils.py#L129
[2] https://github.com/rkansal47/HHbbVV/blob/main/src/run_utils.py#L183
[3] https://github.com/rkansal47/HHbbVV/blob/main/src/condor/submit.templ.sh#L12
[4] https://github.com/rkansal47/HHbbVV/blob/main/src/condor/submit.py#L91